tests/onnx/test_onnx_v2.py

```diff
@@ -208,6 +207,7 @@ def test_values_override(self):
     ("levit", "facebook/levit-128S"),
     ("owlvit", "google/owlvit-base-patch32"),
     ("vit", "google/vit-base-patch16-224"),
+    ("vision-encoder-decoder", "nlpconnect/vit-gpt2-image-captioning"),
```

Comment:
Need to test it using pytest in local environment. However, currently I am having dependency issues on my mac. Would resolve once tested.

Reply:
Yes, it's important we test this before final approval / merging. You can try the slow tests by running:

```
RUN_SLOW=1 pytest tests/onnx/test_onnx_v2.py -k "vision-encoder-decoder"
```

Reply:
Tested with the current version.  Needed to add custom logic in the test_onnx_v2.py, similar to __main__ file.